vtr_flow/scripts/run_vtr_flow.pl

```diff
@@ -1017,6 +1023,168 @@
                 }
             }
         }
+	} elsif ($check_incremental) { # specified channel width && check incremental
```

Comment:
This should probably be a dedicated if statement, independent of the `if(!route_fixed_W) ... else ...`.

As currently written this will actually run VPR 3 times:
1. Initial (pack, place)
2. Check full (pack, place, route)
3. Check incremental (pack, place, route)

I'd suggest re-working it so that when 'check_incremental` is set, it is run separately. So something like:
1. Initial (pack, place, route at fixed W)
2. Check incremental (pack, place, route)

This also keeps the code-paths clearer. At the moment the check_incremental path is embedded in the middle of the 'normal' flow.

I'd also suggest putting the check_incremental into a separate function to further help organize things.